hilda/snippets/macho.py

```diff
@@ -0,0 +1,91 @@
+import lldb
+from construct import *
```

Comment:
i usually use this import style only while developement and debugging, but please import each subcon from `construct` manually. PyCharm should handle it pretty good using `Alt+Enter` on each red name.